terraform/modules/jaspersoft/README.md

```diff
@@ -1,20 +1,23 @@
 # JasperReports Server
 
+The password for the "jasperreports" ldap user must be in a secret named `jasperserver-ldap-bind-password-${var.environment}` using the AWS managed KMS key.
```

Comment:
Update the root README? It only mentions the S3 bucket but should just more generally say to follow any setup instructions here